package.json

```diff
@@ -54,57 +56,50 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-observe-visibility": "^1.0.0",
     "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
     "vuetify": "^2.6.15",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
+    "@originjs/vite-plugin-require-context": "1.0.9",
```

Comment:
There is `require.context` in `CompleteResolution.vue`. This plugin is to fix that.